tests/repr_test.py

```diff
@@ -0,0 +1,43 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright (c) 2022 Scipp contributors (https://github.com/scipp)
+
+import scipp as sc
+import pytest
+
+# TODO: For now, we are just checking that creating the repr does not throw.
```

Comment:
```suggestion
```
... not a useful comment.

Consider changing the test names to include `_does_not_raise` or something?